storage2/src/metrics.rs

```diff
@@ -14,4 +14,13 @@
 pub use metrics::*;
 
 /// Registers all metrics used by this crate.
-pub fn register_metrics() {}
+pub fn register_metrics() {
+    register_gauge!(NCT_SIZE_BYTES);
+    describe_gauge!(
+        NCT_SIZE_BYTES,
+        Unit::Bytes,
+        "The size of the serialized NCT in bytes"
+    );
```

Comment:
I think we should drop this metric; it gives no useful information, because the NCT size is always in a bounded range.